src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -77,12 +77,12 @@ protected function configure()
                 new InputArgument('locale', InputArgument::REQUIRED, 'The locale'),
                 new InputArgument('bundle', InputArgument::OPTIONAL, 'The bundle name or directory where to load the messages'),
                 new InputOption('prefix', null, InputOption::VALUE_OPTIONAL, 'Override the default prefix', '__'),
-                new InputOption('output-format', null, InputOption::VALUE_OPTIONAL, 'Override the default output format', 'xlf'),
+                new InputOption('format', null, InputOption::VALUE_OPTIONAL, 'Override the default output format', 'xlf'),
```

Comment:
This is a BC break, you should keep the old name and emit a deprecation when used.